modules/client/src/client-common/interfaces/common.ts

```diff
@@ -1,5 +1,7 @@
 // This file contains common types, interfaces, and enumerations
 
+import { BigNumberish } from "@ethersproject/bignumber";
```

Comment:
IMO the interfaces we export should be using JS's native data types (Uint8Array). Unless these are only internal for us, and we hand such values to ethers

Reply:
the reason why I would prefer having bignumber is that it can hold bigger numbers than the native javascript types.
For sure we can return the hex string for a number, but that won't help users of the sdk do calculations on it.
The best example are balances of ERC20 tokens where we have 18 decimals for each coin

Reply:
yep as long as you use `n` which informs JS to use BigInt. That is why I refactored it to use BigInt